src/routes/faq/+page.svelte

```diff
@@ -1,5 +1,9 @@
 <svelte:head>
 	<title>FAQ | Kendo Club at the University of Michigan</title>
+	<meta
+		name="description"
```

Comment:
Is this voice typical for descriptions? It feels a little weird for me but I’m not supper familiar with the conventions

Reply:
Resolved in 07baaf88cf3671467bb0b794dc22b87ecdfc9756